src/Types.ts

```diff
@@ -798,7 +835,10 @@ export interface GroupperAPIInternal {
 }
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface GroupperAPI extends GroupperAPIInternal, Disposable {}
+export interface GroupperAPI extends GroupperAPIInternal, Disposable {
+    enterGroupper(element: HTMLElement): HTMLElement | null;
+    escapeGroupper(element: HTMLElement): HTMLElement | null;
```

Comment:
is there any reason for these to be public API since there are events that do these actions?

Reply:
I can actually have one moveFocus(action) method instead of two. It will be more future proof.